lib/connect-dynamodb.js

```diff
@@ -106,7 +106,7 @@ module.exports = function (connect) {
     DynamoDBStore.prototype.get = function (sid, fn) {
 
         sid = this.prefix + sid;
-        var now = +new Date;
+        var now = Math.floor(Date.now() / 1000);
```

Comment:
Locally I modified the test case for the touch method to initialize the session with : expires: Date.now().
After a touch it's converted to seconds correctly. So I believe that recreate the table won't be necessary. .